bin/_mocha

```diff
@@ -112,6 +112,8 @@ program
   .option('--watch-extensions <ext>,...', 'additional extensions to monitor with --watch', list, [])
   .option('--delay', 'wait for async suite definition')
   .option('--allow-uncaught', 'enable uncaught errors to propagate');
```

Comment:
Sorry, I should have made a commit fixing it directly instead of making a comment. Done. Let's see what CI gives us now.